client/src/components/FeedbackForm.js

```diff
@@ -2,22 +2,38 @@
 import React, { useState } from "react";
 import "./feedbackForm.css";
 
-const FeedbackForm = ({ student }) => {
-	console.log("student",student);
-	const [feedbackMessage, setFeedbackMessage] = useState("");
+const FeedbackForm = () => {
+	const [feedbackMessage, setFeedbackMessage] = useState({
+		feedback: "",
+		name: "",
+	});
 	const [givenFeedback, setGivenFeedback] = useState([]);
 	const [feedbackList, setFeedbackList] = useState([]);
 
 	const handleSubmit = (e) => {
 		e.preventDefault();
 		setGivenFeedback(feedbackMessage);
-		setFeedbackList([feedbackMessage, ...feedbackList]);
+		setFeedbackList([feedbackMessage.feedback, ...feedbackList]);
 		setFeedbackMessage("");
+		fetch("/api", {
+			method: "POST",
+			mode: "cors",
+			headers: { "Content-type": "application/json",
+			},
+			body:JSON.stringify(
+				feedbackMessage,
+			),
+		})
+			.then((res) => {
+				res.json(); console.log(res);
+			})
+			.catch((error) => console.log(error));
 	};
 
-	const handleChange = (e) => {
-		setFeedbackMessage(e.target.value);
+	const handleChange = (name) => (e) => {
```

Comment:
@tayfuntayfun Why is name a parameter for handleChange? Also is the syntax correct having two arrows? Could you reconsider having the name here, as it might be omitted if pulled from the DB as per a previous comment?

Reply:
name parameter is to identify each input (as we have multiple inputs to handleChange). It maybe makes more sense if I can add name:"name" HTML attribute to the second input. However not essential since I pass the "name" as parameter to handleChange.  Second question: Curried function. (name, e) is same as "name=>e=>" or (name)=>(e)=>   , all same.